modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi.adoc
+
+[id="csi-drivers-supported_{context}"]
+= CSI drivers supported by {product-title}
+
+{product-title} supports certain CSI drivers that give users storage options that are not possible with in-tree volume plug-ins.
+
+To create CSI-provisioned persistent volumes that mount to these supported storage assets, you can install and configure the CSI driver Operator, which will install the necessary CSI driver and storage class. For more details about installing the Operator and driver, see the documentation for the specific CSI Driver Operator.
+
+The following table describes the CSI drivers that are available with {product-title} and which CSI features they support, such as volume snapshots, cloning, and resize.
```

Comment:
@duanwei33 - Updated wording here for 4.5 only based on the manual Operator install that is required.